src/components/login/Login.test.tsx

```diff
@@ -0,0 +1,12 @@
+import * as enzyme from 'enzyme';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import { Login } from './Login';
+
+describe("Login", () => {
+    const wrapper = enzyme.shallow(<Login client={rest.EvergreenClient("admin", "e4f2c40463dcade5248d36434cb93bac", "http://localhost:8080/api")} />);
+
+    it("matches snapshot", () => {
+        expect(wrapper).toMatchSnapshot();
+    })
+})
```

Comment:
Could you also add tests for clicking the close and submit button? For the submit button, we want to make sure the mock API is called and that the login cookie is stored locally

Reply:
Added tests that check that clicking the close and submit buttons updates the state correctly. I'm still working on figuring out how to check that the cookie is returned and stored correctly.